virttest/ceph.py

```diff
@@ -220,3 +221,65 @@ def create_config_file(ceph_monitor):
         with open(ceph_cfg, 'w+') as f:
             f.write('mon_host = %s' % ceph_monitor)
     return ceph_cfg
+
+
+@error_context.context_aware
+def cephfs_mount(ceph_uri, mount_point, options=None, verbose=False, session=None):
+    """
+    Mount ceph volume to mountpoint.
+
+    :param ceph_uri: str to express ceph uri, e.g 10.73.xx.xx:6789:/
+    :param mount_point: mount point, str
+    :param options: mount options, e.g -o name=admin,secret=AQCAgd5cFyMQLhAAHaz6w+WKy5LvmKjmxx
+    :param verbose: enable verbose or not
+    :param session: mount within the session if given
+    """
+    cephfs_umount(ceph_uri, mount_point, verbose, session)
+    if not os.path.exists(mount_point):
+        try:
+            os.mkdir(mount_point)
+        except OSError as dirError:
+            logging.debug("Creation of the directory:%s failed:%s", mount_point, str(dirError))
+        else:
+            logging.debug("Successfully created the directory %s", mount_point)
```

Comment:
These parts only work on localhost, please update to support the creation/check of mount_point on remote host

Reply:
updated